<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MachineRecipesDialog</class>
 <widget class="QWidget" name="MachineRecipesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add/Edit Recipes for chosen machine</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="lblMachineName">
     <property name="text">
      <string>Recipes for machine: {machine_name}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_contentarea">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="lbl_info">
        <property name="text">
         <string>Recipes</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="tableRecipes">
        <property name="editTriggers">
         <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::SelectionMode::SingleSelection</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Product name</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Capacity (units/hours)</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_buttons">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: #333333;
    color: white;
    border-radius: 4px;
    padding: 6px 12px;
    font-weight: bold;
    font-size: 13px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="btn_addRecipe">
           <property name="text">
            <string>Add Recipe</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_editRecipe">
           <property name="text">
            <string>Edit Recipe</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_deleteRecipe">
           <property name="text">
            <string>Delete Recipe</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
